app/assets/javascripts/directives/project_roles_directive.js

```diff
@@ -0,0 +1,13 @@
+samson.directive('projectRoles', function(userProjectRoleFactory) {
+  return {
+    restrict: 'E',
+    templateUrl: 'shared/_project_roles.tmpl.html',
+    scope: {
+      roles: '=',
+      roleChanged: '&'
+    },
+    link: function($scope, element) {
+      $scope.project_role = userProjectRoleFactory.build(element[0]);
```

Comment:
This looks odd. Not sure what element[0] is supposed to correspond to, the root element of this directive? 
Is there another way to code this to be clearer to read?